### DISCUSSION TITLE
哲学那篇写的毫无逻辑和道理，这玩意不会真有人信吧？

### DISCUSSION THREAD

**Discussion by blesstosam** [The-Run-Philosophy-Organization/run#381] [General]

**Comment by RC-CHN** [comment]
成年人都是固执的 你基本不可能在互联网上说服任何人改变他既有预设的想法

**Comment by letsgow** [comment]
合理的批判我完全支持，但是完全没逻辑的狗屎发言，简直侮辱智商。。。

**Comment by MarionNoel** [comment]
润太祖可是特大人命事故入狱的局长公子，贪二代，有钱的大老板。他大发善心，关爱新一代it耗材，安排咱们世世代代跟着他，给他做贡献。
https://github.com/MarionNoel/juzhanggongzi/discussions/
